support-frontend/assets/components/priceCards/priceCards.tsx

```diff
@@ -109,9 +107,7 @@ export function PriceCards({
 							alternateLabel={`${simpleFormatAmount(
 								currencies[currency],
 								amount,
-							)}${
-								paymentInterval ? amountIntervalSeperator + paymentInterval : ''
-							}`}
+							)}${paymentInterval ? 'per' + paymentInterval : ''}`}
```

Comment:
Should `per` have a few spaces around it? ie ` per `

Reply:
Yeah good catch! 🎣